string NAME=scanned

ObjectDrone d
ObjectArea a
